asv_bench/asv.conf.json

```diff
@@ -50,6 +50,7 @@
         "xlsxwriter": [],
         "xlrd": [],
         "xlwt": [],
+        "pip+odfpy": [],
```

Comment:
Is pip required here?

Reply:
Conda wasn't able to locate the package otherwise(related to `ResolvePackageNotFound`, I think), but I'm not sure whether that's the right fix.

Reply:
Looks like its only on conda-forge. I think better to add that as an available channel to our config:

https://asv.readthedocs.io/en/stable/asv.conf.json.html?highlight=conda#conda-channels

Reply:
Ok, will try that